### PR TITLE
Switch travis scripts to use beaver

### PR DESCRIPTION
**Pull request by mihails-strasuns-sociomantic**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,35 @@
...
+        - DC=dmd-transitional DIST=trusty F=production
+        - DC=dmd-transitional DIST=trusty F=devel
+        - DC=dmd-transitional DIST=xenial F=production
+        - DC=dmd-transitional DIST=xenial F=devel
 
-install: ci/travis-install.sh
+install: beaver install
 
-script: ci/travis-script.sh
+script: beaver run ci/travis.sh
```
Comment (mihails-strasuns-sociomantic) [inline .travis.yml]: @leandro-lucarella-sociomantic how does one pass env vars like `DC` through beaver here?

Comment (mihails-strasuns-sociomantic) [reply .travis.yml]: (right now it is confusingly green because it defaults to dmd1 + production in all matrix branches)

Comment (mihails-strasuns-sociomantic) [reply .travis.yml]: Ping @leandro-lucarella-sociomantic

Comment (leandro-lucarella-sociomantic) [reply .travis.yml]: `BEAVER_DOCKER_VARS="CC DIST"`

I know docs are still pretty bad, but there is something there: https://github.com/sociomantic-tsunami/beaver#docker

**Review by leandro-lucarella-sociomantic — APPROVED** [review]
You might want to use beaver v0.1.2 now. The rest LGTM with the necessary adjustments.

**Comment by mihails-strasuns-sociomantic** [comment]
Fixed env var passing and removed default values to ensure build does not pass by an accident.

**Pull request merged**